test/unit/puppetlabs/services/ca/certificate_authority_core_test.clj

```diff
@@ -483,6 +513,20 @@
         (is (true? (fs/exists? signed-cert-path)))
         (is (= 204 (:status response))))))
```

Comment:
Looks like it's actually only the tests added in this PR that use `/production`, and also since this is against stable probably the new tests probably should be using `/production`, because otherwise the code just thinks that the environment is `/v1`... which doesn't actually matter since it throws out the environment, but is just confusing and kind of wrong.